ivy_tests/test_ivy/test_frontends/test_tensorflow/test_nn.py

```diff
@@ -696,6 +698,7 @@ def test_tensorflow_dropout(
         x=x[0],
         prob=prob,
         scale=scale,
+        noise_shape=noise_shape,
```

Comment:
have to pass in `dtype` argument before `noise_shape` according to function definition